test/audit/audit.rego

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
is there a way we can use the `audit.rego` without needing a full copy of the file under the `test/audit` folder ?

Reply:
yes it works too, but take care to still have `ancestry_path` populated.
Files like in `validator/test/fixture` wont work as the `ancestry_path` is missing